src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsInformation.tsx

```diff
@@ -84,14 +98,9 @@ export const SettingsEditSettingsInformation: React.FC<SettingsEditSettingsInfor
 
             const updatedMe = updateMyUserProfile?.me
 
-            resetForm({
-              values: {
-                name: updatedMe?.name ?? name,
-                email: updatedMe?.email ?? email,
-                phone: updatedMe?.phone ?? phone,
-                password: "",
-              },
-            })
```

Comment:
I am not sure why we need this, If you feel like we can remove it, let's do so. I think there might be a reason to keep it tho

I cannot keep using the `resetForm`, so using `setFieldValue` instead
Can't use `resetForm` because in the response we're not getting the `priceRange`, getting only `priceRangeMin` and `priceRangeMax`. And I can't use `resetForm` with a value missing. In order not to add extra logic I'm not resetting this field

Reply:
probably it's being used to make sure that the formik `dirty` boolean is set to false afterwards - contrarily to using `setField`. If the `dirty` is not being used, this shouldn't be an issue